learn/on-premise/disaster-recovery/Dehydration-and-Rehydration.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Dehydration and Rehydration"
+id: ""
+sidebar_label: "Dehydration and Rehydration"
+---
+---
+
+To Upgrade the operating system OS by migrating Data or for the Disaster and recovery process users can follow the below steps.
```

Comment:
This process is used to rottate instance peridically or dister recovery when data aviable but machines correupted. 
Here data means volumes/direcotries avaiable

Reply:
have to use same verions